software_training_assignment/src/moveTurtle.cpp

```diff
@@ -0,0 +1,46 @@
+/*
+Action server for moving the turtle 
+*/
+
+#include <ros/ros.h> 
+#include <actionlib/server/simple_action_server.h>
+//Subscribe to the geometry_msgs/Twist topic 
+#include <geometry_msgs/Twist.h>
+
+bool moveTurtle(int x, int y){
+    
+
+    return success; 
+}
+
+int main(int argc, char **argv) {
+    ros::init(argc, argv, "moveTurtle"); 
+    ros::NodeHandle nh; 
+
+    //Create the action
+    actionlib::SimpleActionServer<actionlib> as;
+    std::string action_name  = "moveTurtle"; 
+
+    //Create messages to be published to feedback and result
+    actionlib::Feedback feedback; 
+    actionlib::Result result; 
+
+    //Get the pointer to the action goal 
+    const actionlib::GoalConstPtr &goal; 
```

Comment:
not sure what this is, check out the documentation https://wiki.ros.org/actionlib for how to set up an action server